src/Database/LSMTree.hs

```diff
@@ -1393,6 +1391,23 @@ duplicate (Table table) =
 {- |
 Run an action with access to a table that contains the union of the entries of the given tables.
 
+>>> :{
+runExample $ \session table1 -> do
+  LSMT.insert table1 0 "Hello" Nothing
+  LSMT.withTable session $ \table2 -> do
+    LSMT.insert table2 0 "World" Nothing
+    LSMT.insert table2 1 "Goodbye" Nothing
+    LSMT.withUnion table1 table2 $ \table3 -> do
+      print =<< LSMT.lookup table3 0
+      print =<< LSMT.lookup table3 1
+    print =<< LSMT.lookup table1 0
+    print =<< LSMT.lookup table2 0
+:}
+Found (Value "Hello World")
+Found (Value "Goodbye")
+Found (Value "Hello")
+Found (Value "World")
```

Comment:
This example is now duplicated: alternatively, you could only put it in on `withUnion` or `union`

Same is true for other functions that have a `with*` version

Reply:
They should be different in that one version uses bracket.

Reply:
Either way, whichever function you look at should have an example.